### ISSUE TITLE
Mingw csources path error

### ISSUE THREAD

**Issue by VVX7** [status-im/nim-bearssl#13]
Cross-compiling fails on a Ubuntu 20.04 host to a Windows target.  Compiling any program that imports nim-bearssl results in the following error where mingw does not understand the csources path.

`nim c -d:mingw --threads:on test.nim`

```
/home/user/.nimble/pkgs/libp2p-0.0.2/libp2p/peerinfo.nim(66, 21) Hint: 'init' cannot raise 'Defect' [XCannotRaiseY]
CC: decls.nim\csources\src\codec\ccopy
CC: decls.nim\csources\src\codec\dec16be
CC: decls.nim\csources\src\codec\dec16le
CC: decls.nim\csources\src\codec\dec32be
x86_64-w64-mingw32-gcc: error: /home/user/.nimble/pkgs/bearssl-0.1.5/bearssl/decls.nim\csources\src\codec\ccopy.c: No such file or directory
x86_64-w64-mingw32-gcc: fatal error: no input files
compilation terminated.
CC: decls.nim\csources\src\codec\dec32le
CC: decls.nim\csources\src\codec\dec64be
Error: execution of an external compiler program '/usr/bin/x86_64-w64-mingw32-gcc -c  -w -fmax-errors=3 -mno-ms-bitfields -DWIN32_LEAN_AND_MEAN -I/home/user/.nimble/pkgs/bearssl-0.1.5/bearssl/decls.nim\csources\src -I/home/user/.nimble/pkgs/bearssl-0.1.5/bearssl/decls.nim\csources\inc -I/home/user/.nimble/pkgs/bearssl-0.1.5/bearssl/decls.nim\csources\tools -DBR_USE_WIN32_TIME=1 -DBR_USE_WIN32_RAND=1 -DBR_LE_UNALIGNED=1 -DBR_64=1  -DBR_amd64=1 -DBR_INT128=1 -I/home/user/.nimble/pkgs/secp256k1-0.5.1/secp256k1_abi.nim\secp256k1_wrapper -I/home/user/.nimble/pkgs/secp256k1-0.5.1/secp256k1_abi.nim\secp256k1_wrapper\secp256k1 -I/home/user/.nimble/pkgs/secp256k1-0.5.1/secp256k1_abi.nim\secp256k1_wrapper\secp256k1\src -DHAVE_CONFIG_H -DHAVE_BUILTIN_EXPECT   -I/usr/local/lib/nim -I/home/user/Nim/test -o '/home/user/.cache/nim/test_libp2p_d/decls.nim\csources\src\codec\ccopy.c.o' '/home/user/.nimble/pkgs/bearssl-0.1.5/bearssl/decls.nim\csources\src\codec\ccopy.c'' failed with exit code: 1
```

**Comment by jangko** [comment]
well, actually it is a `currentSourcePath` or `instantiationInfo` bug. they are not aware about cross compiling path separator difference.

**Comment by VVX7** [comment]
Thanks for opening that issue!

**Comment by jangko** [comment]
fixed by #15.

@VVX7, also you need to `git submodule init` and `git submodule update` if you `git clone` this repo.
